test/bin/build_images.sh

```diff
@@ -197,7 +197,8 @@ do_group() {
     local blueprint_file
     local build_name
     local buildid
-    local buildid_list=""
+    local buildid_list=()
+    local builds_to_get=""
```

Comment:
Nit: this declaration can be moved down to the scope where it's used.

Reply:
Yeah, but it seems that we went with declaring them at the top of function so didn't want to diverge from the style